src/buildercore/terraform.py

```diff
@@ -445,6 +477,12 @@ def add_table(table_id, table_options):
     if not tf_file['resource']['google_bigquery_table']:
         del tf_file['resource']['google_bigquery_table']
 
+    if not tf_file['data'][DATA_TYPE_HTTP]:
```

Comment:
have a refactoring for this setting/unsetting, which was a preexisting bug; will do it on another PR as this one is already big. If I manage to post this comment without Github falling over